### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by area363**
This PR reflects the most recent updates in NineChronicles.Headless

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ore-path <String>] [--ice-server <String>...] [--peer <String>...] [--no-trusted
 --graphql-host <String>] [--graphql-port <Nullable`1>] [--libplanet-node] [--workers <Int32>] [--confirmations <Int32>] [--max-transactions <Int32>] [--strict-rendering] [--dev] [--dev.block-interval <Int32>] [--dev.reorg-interval <Int32>] [--log-m
 inimum-level <String>] [--aws-cognito-identity <String>] [--aws-access-key <String>] [--aws-secret-key <String>] [--aws-region <String>] [--help] [--version]
 
-Run standalone application with options.
+Run headless application with options.
```
Comment (moreal) [inline README.md:13]: You MUST update the string in [Program.cs#L48](https://github.com/planetarium/NineChronicles.Headless/blob/a86c323caf2c8f2058590e36371f041f40a90737/NineChronicles.Headless.Executable/Program.cs#L48)  if you want update this line.

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Options:
   --dev                                                    Flag to turn on the dev mode.  false by default.
   --dev.block-interval <Int32>                             The time interval between blocks. It's unit is milliseconds. Works only when dev mode is on.  10000 (ms) by default. (Default: 10000)
   --dev.reorg-interval <Int32>                             The size of reorg interval. Works only when dev mode is on.  0 by default. (Default: 0)
-  --log-minimum-level <String>                             The log minimum level during standalone execution. (Default: debug)
+  --log-minimum-level <String>                             The log minimum level during headless execution. (Default: debug)
```
Comment (moreal) [inline README.md:43]: You MUST update the string in [Program.cs#L114](https://github.com/planetarium/NineChronicles.Headless/blob/a86c323caf2c8f2058590e36371f041f40a90737/NineChronicles.Headless.Executable/Program.cs#L114) if you want update this line.

**Review by moreal — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -125,14 +125,14 @@ Usage: docker volume create [<VOLUME_NAME>]
...
 --volume 9c-volume:/app/data \
-planetariumhq/ninechronicles-headless \
-<a href = "#run" title="NineChronicles Standalone options">[NineChronicles Standalone Options]</a>
+planetariumhq/ninechronicles-headless:latest \
+<a href = "#run" title="NineChronicles Headless options">[NineChronicles Headless Options]</a>
 </pre>
 #### Note)
-* Instead of including the "--ice-server" option, allocate an Elastic IP to your instance and include it as the "--host" option and after adding an inbound port, include it as the "--port" option. Refer to these official docs on [Elastic IP allocation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) and [editing inbound rules](https://docs.aws.amazon.com/cli/latest/userguide/cli-services-ec2-sg.html) for more information.
-* For mining, make sure to include "--private-key" option with your private key. Also, include "--libplanet-node" to run the default libplanet node. 
+* If you want to use the same headless options as your game client, refer to **`config.json`** under **`%localappdata%\Programs\Nine Chronicles\resources\app`**.
```
Comment (moreal) [inline README.md:133]: I'm not sure to comment game client here. And it doesn't comment what the game means.

File: README.md [hunk 4]
```diff
@@ -125,14 +125,14 @@ Usage: docker volume create [<VOLUME_NAME>]
...
+* If you are using an [Elastic IP](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) on your AWS instance, you do not need to include the `--ice-server` option.
```
Comment (moreal) [inline README.md:134]: Is it okay not to comment other parameters (i.e., `--host` and `--port`) commented in README before changed?

File: README.md (hunk 1 above)
Comment (area363) [reply README.md:13]: I see, I'll put a PR for that update!

File: README.md (hunk 2 above)
Comment (area363) [reply README.md:43]: I see, I'll put a PR for that update!

File: README.md (hunk 3 above)
Comment (area363) [reply README.md:133]: Is it better to say "Nine Chronicles" than "game client"? I assumed that users that would want to run NineChronicles.Headless have already played the game so they would know what "game client" means.

File: README.md (hunk 4 above)
Comment (area363) [reply README.md:134]: After removing `--host` and `--port` args in Dockerfile(https://github.com/planetarium/NineChronicles.Headless/pull/206/files), users no longer need to include those parameters to run the headless application on AWS so I removed those comments. I suppose I can add the description on `--host` and `--port` if users were to use an Elastic IP.

Comment (moreal) [reply README.md:134]: IMO it should comment more before removing it. Libplanet requires either `--ice-server` and `--host`.

```csharp
            if (_host == null && (_iceServers == null || !_iceServers.Any()))
            {
                throw new ArgumentException(
                    $"Swarm requires either {nameof(host)} or " +
                    $"{nameof(iceServers)}."
                );
            }
```

Also the option, `--host`'s default value is `null` so it should comment about the option.

**Review by area363 — COMMENTED** [review]
Suggestions applied.

**Review by moreal — CHANGES_REQUESTED** [review]
~~It seems that the code has been not applied yet. [Program.cs#L48](https://github.com/planetarium/NineChronicles.Headless/blob/a86c323caf2c8f2058590e36371f041f40a90737/NineChronicles.Headless.Executable/Program.cs#L48) and [Program.cs#L114](https://github.com/planetarium/NineChronicles.Headless/blob/a86c323caf2c8f2058590e36371f041f40a90737/NineChronicles.Headless.Executable/Program.cs#L114)~~

It should be merged after https://github.com/planetarium/NineChronicles.Headless/pull/223 merged

**Review by limebell — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (limebell) [inline README.md:43]:
```suggestion
  --log-minimum-level <String>                             The log minimum level during headless execution.  debug by default. (Default: debug)
```
This line does not match with https://github.com/planetarium/NineChronicles.Headless/blob/7bbf8d99b682ec9cf8ca39518022f43f390cf2e2/NineChronicles.Headless.Executable/Program.cs#L112

**Review by limebell — CHANGES_REQUESTED** [review]

**Comment by moreal** [comment]
It seems staled.

**Review by riemannulus — APPROVED** [review]

**Comment by area363** [comment]
> It seems staled. Could you rebase this pull request?

@moreal I just resolved the conflicts. Can you review it again? 😃

**Review by moreal — APPROVED** [review]

**Review by limebell — APPROVED** [review]

**Pull request merged**